docs/wiki/deployment/file-integrity-monitoring.md

```diff
@@ -85,15 +87,29 @@ fs.inotify.max_queued_events = 32768
 
 ## File Accesses
 
-File accesses on Linux using inotify may induce unexpected and unwanted performance reduction. To prevent 'flooding' of access events alongside FIM, access events for `file_path` categories is an explicit opt-in. Add the following list of categories:
+In addition to FIM which generates events if a file is created/modified/deleted, osquery also supports file access monitoring which can generate events if a file is accessed.
+
+File accesses on Linux using inotify may induce unexpected and unwanted performance reduction. To prevent 'flooding' of access events alongside FIM, enabling access events for `file_path` categories is an explicit opt-in. You may add categories that were defined in your `file_paths` stanza:
 
 ```json
 {
+  "file_paths": {
+    "homes": [
+      "/root/.ssh/%%",
+      "/home/%/.ssh/%%"
+    ],
+    "etc": [
+      "/etc/%%"
+    ],
+    "tmp": [
+      "/tmp/%%"
+    ]
+  },
   "file_accesses": ["homes", "etc"]
 }
 ```
 
-To enable access monitoring for the above set of directories in 'homes' and the single 'etc'.
+The above configuration snippet will enable file integrity monitoring for 'homes', 'etc', and 'tmp' but only enable access monitoring for the above set of 'homes' and the single 'etc' directories.
```

Comment:
you say `single 'etc' directories`, but the globbing recurses, maybe this should just be worded `and 'etc' directories`?